cms/templates/studio_xblock_wrapper.html

```diff
@@ -76,7 +80,12 @@
                     % if can_edit:
                         % if not show_inline:
                             <li class="action-item action-edit">
-                                <button class="btn-default edit-button action-button">
+                                <button class="btn-default edit-button action-button"
+                                    use-new-editor-text = ${use_new_editor_text}
+                                    use-new-editor-video = ${use_new_editor_video}
+                                    use-new-editor-problem = ${use_new_editor_problem}
+                                    authoring_MFE_base_url = ${get_editor_page_base_url(xblock.location.course_key)}
+                                >
```

Comment:
Does the addition of the flags here mean that to enable/disable the feature by performing both things below?:
- flipping the waffle flag
- deploying a new edx-platform release

This page is a statically built page, correct? If so, I think both things *will* be needed.

Reply:
You make an excellent point here. This poses:
a. A bunch of risk (we have to wait an hour to turn the feature off if something is broken)
b. a minor annoyance to turning it on (especially if we do so incrementally which is what you suggested above, and I agree with)
I spent about a week trying to find a way to this without passing the flag state into the html template, but I'm open to suggestions, and I think your point makes it difficult to NOT search for another solution right now.

Reply:
The only other way I could think to do this would be to add some field to the xblock, but that would require a dat migration for all blocks every time we want to set the flag, which is arguably worse. Maybe we only release this when the editors are really set up nice.

Reply:
@ormsbee Would love some of your advice here. Is there a way we could pass along the state of a waffle flag when we serve the xblock to studio, and then somehow detect that state in our javascript. If you don't know, thats fine too! Thanks so much!

Reply:
Ok so, I have an addendum to make here:
Going back and looking at my testing plan you'll notice that I didn't include the need to remake static assets after changing the flags, and I know I followed my test plan to a T. Only looking at how the mako template pulls in xblock data I realized something: You don't need to do a static build every time the flag changes. this is because this code is executed in a 
`<% %>` block, not a `<%! %>` block. That `!` indicates that that version should be complied statically. The method goes and checks the status of the waffle flag every time a page is served. You can test it for yourself. I'll point to the code where that happens. in another comment